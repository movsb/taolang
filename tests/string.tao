function main() {
    index();
    lower();
}

function index() {
    let s = "我1A们";
    println(s[0], s[1], s[2], s[3]);
    println(s.len());
}

function lower() {
    println("AbC".lower());
}
