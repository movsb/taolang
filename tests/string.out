我 1 A 们
4
abc
